# ExampleMessage.msg
# A simple message template for ROS

int32 id          # Unique identifier for the message
string name       # A name or label
float64 value     # Some numeric value
